### PR TITLE
PAYARA-1958 Propertise the versions within the poms

### PR DESCRIPTION
**Pull request by michaelranaldo**

### CODE REVIEW

**Comment by michaelranaldo** [comment]
Jenkins test please

File: appserver/pom.xml [hunk 1]
```diff
@@ -543,11 +560,11 @@
...
             </dependency>
-	    <dependency>
-		<groupId>javax.resource</groupId>
-		<artifactId>javax.resource-api</artifactId>
-		<version>${javax.resource-api.version}</version>
-	    </dependency>
+      	    <dependency>
+      		      <groupId>javax.resource</groupId>
+      		        <artifactId>javax.resource-api</artifactId>
+      		          <version>${javax.resource-api.version}</version>
```
Comment (Pandrex247) [inline appserver/pom.xml]: Can you fix all this weird formatting please?

**Comment by michaelranaldo** [comment]
Jenkins test please

**Comment by Pandrex247** [comment]
@michaelranaldo this had some conflicts, can you double check what I didn't kill anything please? https://github.com/payara/Payara/pull/1868/commits/bc98c3987315e000123893abfba65b4f0307523b

**Comment by michaelranaldo** [comment]
Looks okay - fixed one remaining conflict but otherwise seems good. A few whitespaces have crept in however...

**Comment by michaelranaldo** [comment]
Jenkins test please

**Pull request merged**